### ISSUE TITLE
Add support for SSL

### ISSUE THREAD

**Issue by martindsouza** [martindsouza/docker-ords#7]

**Comment by rimblas** [comment]
Yes please. Definitely desired as there are more and more obstacles to mixing secure and non-secure sources.

**Comment by pullandfork** [comment]
How would this work? Is this something at the host level or container? The web server in ORDS is jetty, does that support SSL?

Sorry I see Tim has already addressed this

https://oracle-base.com/articles/misc/oracle-rest-data-services-ords-standalone-mode
